.doc_gen/metadata/services.yaml

```diff
@@ -154,11 +214,26 @@ s3:
   short: "&S3;"
   sort: "S3"
   version: s3-2006-03-01
+sagemaker:
+  long: "&SMlong;"
+  short: "&SM;"
+  sort: "SageMaker"
+  version: sagemaker-2017-07-24
 secrets-manager:
   long: "&ASMlong;"
   short: "&ASM;"
   sort: "Secrets Manager"
   version: secretsmanager-2017-10-17
+ses:
+  long: "&SESlong;"
+  short: "&SES;"
+  sort: "SES"
+  version: sesv2-2019-09-27
```

Comment:
Assuming you want SES v2, I think we need another couple entities added to phrases-code-examples.ent:
```
<!ENTITY SESv2long "&SESlong; API v2">
<!ENTITY SESv2 "&SES; API v2">
```

Reply:
I'll create another PR for the new entities.

Reply:
Fixed.